src/clj/game/core/payment.clj

```diff
@@ -10,6 +10,7 @@
 
 (defmulti cost-name (fn [[cost-type _]] cost-type))
 (defmulti value (fn [[cost-type _]] cost-type))
+(defmulti stealth-value (fn [[cost-type _ __]] cost-type))
```

Comment:
Oh shit I'm a fool lmao. Wish I knew how to read. You still should make a default handler here but this is obviously the defmulti.

Reply:
I've added the default, but I'm not sure what you mean about the defmulti. It's the defmulti you're commenting on. Maybe this is something about clojure I don't know, if so you're gonna have to prod me in the right direction.

Reply:
Are you on stimslack? If not, join here https://stimslackinvite.herokuapp.com/ and then get into the #jinteki_net channel so we can talk instead of just commenting on github.